Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.cpp

```diff
@@ -67,7 +165,11 @@ namespace AZ::DocumentPropertyEditor
                         return;
                     }
 
-                    if (name == Reflection::DescriptorAttributes::Label || name == Reflection::DescriptorAttributes::Handler)
+                    const AZStd::array ignoredAttributes = { Reflection::DescriptorAttributes::Label,
+                                                             Reflection::DescriptorAttributes::Handler,
+                                                             Reflection::DescriptorAttributes::Container,
+                                                             Nodes::PropertyEditor::Visibility.GetName() };
+                    if (AZStd::find(ignoredAttributes.begin(), ignoredAttributes.end(), name) != ignoredAttributes.end())
```

Comment:
FYI you can use AZStd::ranges::find to search over the array
```suggestion
                    if (AZStd::ranges::find(ignoredAttributes. name) != ignoredAttributes.end())
```

Reply:
Ooh I didn't know you'd done a pass on a range implementation! What a great step towards solving one of my absolute least favorite things about STL's API - much appreciated, sir.